test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -22,28 +35,119 @@ Describe "Job Cmdlet Tests" -Tag "CI" {
                 $_.FullyQualifiedErrorId | should be "JobWithSpecifiedNameNotFound,Microsoft.PowerShell.Commands.GetJobCommand"
             }
         }
-        It "Receive-Job can retrieve job results" -pending {
+        It "Receive-Job can retrieve job results" {
             $waitjob = Wait-Job -Timeout 10 -id $j.id
             if ( $waitJob ) {
                 $result = receive-job -id $j.id
-                $result.id | Should be 2
+                $result | Should be 2
             }
         }
     }
     Context "jobs which take time" {
+        BeforeAll {
+            $jobBlock =
```

Comment:
why do you want to replace the original `{ start-sleep 15 }` with this script block? None of the tests in this Context check the output of the job, so it's not necessary to write out anything. If you think 15 seconds is more than needed, maybe just `{ start-sleep 10 }`?